src/main/scala/chisel3/stage/ChiselStage.scala

```diff
@@ -0,0 +1,30 @@
+// See LICENSE for license details.
+
+package chisel3.stage
+
+import firrtl.AnnotationSeq
+import firrtl.options.{Phase, Shell, Stage}
+import firrtl.options.Viewer.view
+import firrtl.stage.{FirrtlCli, FirrtlStage}
+
+class ChiselStage extends Stage {
+  val shell: Shell = new Shell("chisel") with ChiselCli with FirrtlCli
+
+  private val phases: Seq[Phase] =
+    Seq( new chisel3.stage.phases.Checks,
+         new chisel3.stage.phases.Elaborate,
+         new chisel3.stage.phases.AddImplicitOutputFile,
+         new chisel3.stage.phases.AddImplicitOutputAnnotationFile,
+         new chisel3.stage.phases.Emitter,
+         new chisel3.stage.phases.Convert,
+         new chisel3.stage.phases.MaybeFirrtlStage )
+      .map(firrtl.options.phases.DeletedWrapper(_))
+
+  def run(annotations: AnnotationSeq): AnnotationSeq = {
+    val cOpts = view[ChiselOptions](annotations)
```

Comment:
There's a corollary here that you may have caught onto. The deeper that I got into migrating to phases, the less and less that the views were needed. If you wind up needed a common view of some set of options, it likely indicates that your phases are not suitably decoupled. There are exceptions, e.g., `TargetDirAnnotation` is reasonable for everybody to view, but this may be an anti-pattern to watch out for. The view concept predates Stage/Phase back when the Drivers did a lot.

Reply:
Fixed in rebase.